CHANGELOG.md

```diff
@@ -14,10 +16,6 @@
 * BraintreeCore
   * Fix analytics bug where sessionID value in analytics payload was inaccurate; send separate FPTI POST requests per unique sessionID
 
-## 7.0.0-beta1
-* Breaking Changes
-  * Bump minimum supported deployment target to iOS 16+
```

Comment:
We also want to preserve this existing changelog entry

Reply:
addressed in c386bf3f3eca0db14967a3da1611e2c0f6310230, moved it to the top